sap_business_document_processing/document_classification_client/dc_api_client.py

```diff
@@ -0,0 +1,472 @@
+        return response.json()[API_MODELS_FIELD]
+
+    def get_trained_model_info(self, model_name, model_version):
+        """
+        Gets information about a specific trained model
+        :param model_name: The name of the model
+        :param model_version: The version  of the model
+        :return: Object containing the training status and training accuracy data
+        """
+        response = self.get(TRAINED_MODEL_ENDPOINT(model_name=model_name, model_version=model_version),
+                            log_msg_before=f'Getting information about the model {model_name} with version {model_version}',
+                            log_msg_after=f'Successfully got the information about the model {model_name} with version {model_version}')
+        return response.json()
+
+    # Model deployment
+    def deploy_model(self, model_name, model_version):
+        """
+        Deploys a trained model to be available for inference
+        :param model_name: The name of the trained model
+        :param model_version: The version of the trained model
+        :return: Object containing information about the deployed model serving
+        """
+        response = self.post(DEPLOYMENTS_ENDPOINT, validate=False,
+                             json={
+                                 API_MODEL_VERSION_FIELD: model_version,
+                                 API_MODEL_NAME_FIELD: model_name
+                             },
+                             log_msg_before=f'Triggering the deployment of the model {model_name} with version {model_version}')
+        if response.status_code == 409:
+            # TODO: Change the API to differ between the 409 codes, see: DIGITALCONTENTPROCESSING-709
+            if 'model is already deployed' in response.text:
+                self.raise_for_status_with_logging(response)
+            time.sleep(self.polling_long_sleep)
+            return self.deploy_model(model_name, model_version)
+        self.raise_for_status_with_logging(response)
+        return self._poll_for_url(MODEL_DEPLOYMENT_ENDPOINT(deployment_id=response.json()[API_DEPLOYMENT_ID_FIELD]),
+                                  log_msg_before=f'Successfully triggered the deployment of the model {model_name} '
+                                  f'with version {model_version}, waiting for the deployment completion',
+                                  wait_status=409,
+                                  sleep_interval=self.polling_long_sleep).json()
+
+    def get_deployed_models_info(self):
+        """
+        Gets information about all deployed model servings
+        :return: An array of all deployed model servings
+        """
+        response = self.get(DEPLOYMENTS_ENDPOINT,
+                            log_msg_before='Getting information about all deployed models',
+                            log_msg_after='Successfully got information about all deployed models')
+        return response.json()[API_DEPLOYMENTS_FIELD]
```

Comment:
Yes, I think a notice, that there is a new library and that the old one will not be developed further is very important. I will collect all the breaking differences so that we can include a quick guide for changing the libraries as well.